src/main/java/org/openjdk/shenandoah/DataLogProvider.java

```diff
@@ -73,6 +74,8 @@ public class DataLogProvider {
     private int snapshotsIndex = -1;
     private Snapshot currSnapshot = DISCONNECTED;
 
+    private long LATEST_VERSION = 2;
```

Comment:
Should be `static final` and we can use this instead of the `2` argument for `DISCONNECTED` `Snapshot` constructor.